src/js/components/Graphs/graph.jsx

```diff
@@ -34,15 +38,24 @@ const selectors = {
   GRAPH: '.graphContainer .graph',
 };
 
+const displayOptions = {
+  graph: 'graph',
+  performanceTimeLine: 'performanceTimeLine',
+};
+
 const GraphComponent = ({
   graphJson,
   fragmentId,
 }) => {
+  const [displayOption, setDisplayOption] = useState(displayOptions.graph);
+
   useEffect(() => {
     if (graphJson) {
       const graphDeclaration = constructGraph(graphJson);
       const network = drawGraph(graphDeclaration, document.querySelector(selectors.GRAPH));
```

Comment:
I believe you can add ref on line 79 and use it here.

Reply:
draw graph pass it to vis function. Im not sure vis handle react ref